# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xalan-c/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur xml-xalan-orig/c/src/xalanc/PlatformSupport/DirectoryEnumerator.hpp xml-xalan/c/src/xalanc/PlatformSupport/DirectoryEnumerator.hpp
--- xml-xalan-orig/c/src/xalanc/PlatformSupport/DirectoryEnumerator.hpp	2009-02-09 01:36:13.000000000 +0000
+++ xml-xalan/c/src/xalanc/PlatformSupport/DirectoryEnumerator.hpp	2009-02-09 01:58:01.000000000 +0000
@@ -33,7 +33,7 @@
 #endif
 
 
-
+#include <cstring>
 #include <functional>
 #include <iterator>
 
diff -Nur xml-xalan-orig/c/src/xalanc/XMLSupport/FormatterToHTML.cpp xml-xalan/c/src/xalanc/XMLSupport/FormatterToHTML.cpp
--- xml-xalan-orig/c/src/xalanc/XMLSupport/FormatterToHTML.cpp	2009-02-09 01:36:13.000000000 +0000
+++ xml-xalan/c/src/xalanc/XMLSupport/FormatterToHTML.cpp	2009-02-09 01:43:14.000000000 +0000
@@ -26,7 +26,7 @@
 
 
 #include <cassert>
-
+#include <cstring>
 
 
 #include <xercesc/sax/AttributeList.hpp>
diff -Nur xml-xalan-orig/c/src/xalanc/XSLT/ElemNumber.cpp xml-xalan/c/src/xalanc/XSLT/ElemNumber.cpp
--- xml-xalan-orig/c/src/xalanc/XSLT/ElemNumber.cpp	2009-02-09 01:36:13.000000000 +0000
+++ xml-xalan/c/src/xalanc/XSLT/ElemNumber.cpp	2009-02-09 01:47:46.000000000 +0000
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 #include "ElemNumber.hpp"
-
+#include <cstring>
 
 
 #include <xercesc/sax/AttributeList.hpp>
diff -Nur xml-xalan-orig/c/src/xalanc/XalanDOM/XalanDOMString.cpp xml-xalan/c/src/xalanc/XalanDOM/XalanDOMString.cpp
--- xml-xalan-orig/c/src/xalanc/XalanDOM/XalanDOMString.cpp	2009-02-09 01:36:13.000000000 +0000
+++ xml-xalan/c/src/xalanc/XalanDOM/XalanDOMString.cpp	2009-02-09 01:37:18.000000000 +0000
@@ -26,7 +26,7 @@
 
 
 #include <cstdlib>
-
+#include <string.h>
 
 
 XALAN_CPP_NAMESPACE_BEGIN
diff -Nur xml-xalan-orig/c/src/xalanc/XalanExe/XalanExe.cpp xml-xalan/c/src/xalanc/XalanExe/XalanExe.cpp
--- xml-xalan-orig/c/src/xalanc/XalanExe/XalanExe.cpp	2009-02-09 01:36:13.000000000 +0000
+++ xml-xalan/c/src/xalanc/XalanExe/XalanExe.cpp	2009-02-09 02:03:52.000000000 +0000
@@ -17,7 +17,7 @@
 #include <xalanc/Include/PlatformDefinitions.hpp>
 
 
-
+#include <cstring>
 #include <cstdlib>
 #if defined(XALAN_CLASSIC_IOSTREAMS)
 #include <iostream.h>
